bindings/qt/sdk.pri

```diff
@@ -176,16 +204,21 @@ CONFIG(USE_MEDIAINFO) {
 CONFIG(USE_LIBRAW) {
     DEFINES += HAVE_LIBRAW
 
+    vcpkg:LIBS += -lraw$$DEBUG_SUFFIX -ljasper$$DEBUG_SUFFIX
+    vcpkg:win32:LIBS += -ljpeg$$DEBUG_SUFFIX
+    vcpkg:!win32:LIBS += -ljpeg
+    vcpkg:unix:!macx:LIBS += -lgomp
```

Comment:
without pdfium this requires `-llcms2$$DEBUG_SUFFIX`

Reply:
thanks, added